cpm

```diff
@@ -6,18 +6,21 @@ phi() {
 }
 
 usage() {
-    >&2 echo "cpm [i|r|l|C|u|U|s|S|I|c|h] [pkg]..."
-    phi i "install" "install one or more packages"
-    phi r "remove " "remove one or more packages"
-    phi l "list   " "list installed packages"
-    phi C "count  " "count installed packages"
-    phi u "update " "update package lists"
-    phi U "upgrade" "upgrade all packages"
-    phi s "search " "search for a package"
-    phi S "show   " "show information about a package"
-    phi I "info   " "same as show"
-    phi c "clean  " "clean up leftover files/caches/orphans"
-    phi h "help   " "show this message"
+    >&2 echo "cpm [i|r|l|C|u|U|s|S|I|f|c|h] [pkg]..."
+    phi i  "install" "install one or more packages"
+    phi r  "remove " "remove one or more packages"
+    phi l  "list   " "list installed packages"
+    phi C  "count  " "count installed packages"
+    phi u  "update " "update package lists"
+    phi U  "upgrade" "upgrade all packages"
+    phi s  "search " "search for a package"
+    phi S  "show   " "show information about a package"
+    phi I  "info   " "same as show"
+    phi \  "files  " "show file list of package"
+    phi \  "from   " "show package which owns a file"
+    phi f  "       " "show file list / package which owns file"
```

Comment:
Consider putting the "f" on "from" and "files" and then removing the third line. Just make it implied that this is how it works.

Reply:
Alright.